libraries/AP_OSD/AP_OSD.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * This file is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#include <AP_Param/AP_Param.h>
+
+class AP_OSD_Backend;
+
+#define AP_OSD_NUM_SCREENS 4
```

Comment:
4 screens as default seems excessive, especially since currently only 1 is being used. I suggest we change this to 1

Reply:
we'll support 4 soon